core/environment.go

```diff
@@ -54,6 +56,18 @@ func (env *Env) SetEnvArgs(newArgs []string) {
 	}
 }
 
+func (env *Env) SetClassPath(cp string) {
+	cparray := filepath.SplitList(cp)
```

Comment:
Here and below, names should be camel case: `cpArray` etc.

Reply:
Fixed.